modules/ROOT/pages/storage.adoc

```diff
@@ -354,6 +354,70 @@ storage:
       with_mount_unit: true
 ----
 
+== Adding Network Storage
+
+Fedora CoreOS systems can be configured to mount network file systems such as NFS and CIFS. This is best achieved by using Ignition to create systemd units. File systems can be mounted on boot by creating a standard mount unit. Alternately, a file system can be mounted when users access the mountpoint by creating an additional automount unit. Below are examples of each for an NFS file system.
+
+=== Configuring NFS Mounts
+
+Creating a systemd unit to mount an NFS file system on boot. Note: the .mount file must be named based on the path (e.g. /var/mnt/data = var-mnt-data.mount)
+
+[source,yaml]
+----
+variant: fcos
+version: 1.3.0
+systemd: 
+  units:
+    - name: var-mnt-data.mount
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Mount data directory
+
+        [Mount]
+        What=example.server.edu:/data
```

Comment:
`example.com` (or org, or net) is [reserved](https://www.rfc-editor.org/rfc/rfc2606.html) for this sort of use.

Reply:
Switched to example.org